app/test_main.py

```diff
@@ -1,3 +1,68 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age",
+    [
+        pytest.param(
+            0, 0, [0, 0],
+            id="Should return zero if cat or dog age equal to zero"
+        ),
+        pytest.param(
+            14, 14, [0, 0],
+            id="Should return zero if cat or dog age less than 15"
+        ),
+        pytest.param(
+            15, 15, [1, 1],
+            id="Should return 1 if cat or dog age is 15"
+        ),
+        pytest.param(
+            23, 23, [1, 1],
+            id="Should return one if cat or dog "
+               "age more than 14 and less than 24"
```

Comment:
The test case description is not entirely accurate. It should specify that the function should return 1 when the age is between 15 and 23, inclusive. Consider updating the description for clarity.